Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -23,9 +23,9 @@ public LibraryService(IDbConnection connection)
             databaseConnection = connection;
         }
 
-        public IEnumerable<Book> GetBooks()
+        public IEnumerable<Book> GetBooks(string search)
         {
-            return databaseConnection.Query<Book>("SELECT * FROM Books");
+            return databaseConnection.Query<Book>($"SELECT * FROM Books WHERE Books.title LIKE '%{search}%' OR Books.authors LIKE '%{search}%'");
```

Comment:
You should use dapper parameters: https://dapper-tutorial.net/parameter-anonymous, so that you're protected against [SQL Injection](https://www.learndapper.com/parameters#what-is-sql-injection) 🙂